tests/model_geometry_test.js

```diff
@@ -0,0 +1,486 @@
+'use strict';
+
+require('./helper');
+
+const { expect } = require('chai'),
+  { DataTypes } = require('../source'),
+  dialect = 'postgres',
+  semver = require('semver');
+
+const version_helper = require('../source/version_helper.js')
+const { makeTestSequelizeInstance } = require('./helper.js')
+
+function maybeSkip(test, isCRDBVersion21_2Plus) {
```

Comment:
nit: maybe even rename the param here to `shouldSkip`

Reply:
Done